pdsc/ingest.py

```diff
@@ -172,11 +180,20 @@ def store_segments(outputfile, metadata, config):
     progress = standard_progress_bar('Segmenting footprints')
     for m in progress(metadata):
         try:
-            s = TriSegmentedFootprint(m, resolution, localizer_kwargs)
+            # erd: if not Mars, store body radius
+            if body_radius == MARS_RADIUS_M:
+                s = TriSegmentedFootprint(m, resolution, localizer_kwargs)
+            else:
+                s = TriSegmentedFootprint(m, resolution, localizer_kwargs, body_radius)
```

Comment:
Can remove the `if`/`else` block and just always pass in `body_radius=body_radius` argument.